### ISSUE TITLE
Migrate notifications telemetry

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/focus-android#5651]
Breakout from #5488.

> Question being answered by the data:

How often are users using current notifications from Focus?

**Comment by mcarare** [comment]
Note: This should also include the telemetry for Shortcuts, at the moment some of the old telemetry does not differentiate between the same actions(from notification or shortcuts)

**Comment by lobontiumira** [comment]
Verified on the latest Focus Nightly 95.0 (build 353151018 GV 96.0a1-20211110092453) with Google Pixel (Android 10).
The events from the notification were generated successfully, but the ones from the app's shortcut are not - I've filed #5851.
@mcarare I'll remove the QAReady label for now.
 

Glean ping to URL: /submit/org-mozilla-focus-nightly/events/1/9ccb7bf3-2f06-49b2-9d38-441b139c1315
```
{
          "timestamp": 0,
          "category": "notifications",
          "name": "erase_open_button_tapped",
          "extra": {
            "opened_tabs": "1"
          }
        }
```
```
{
          "timestamp": 6008,
          "category": "notifications",
          "name": "open_button_tapped"
        }
```
```
{
          "timestamp": 8751,
          "category": "notifications",
          "name": "notification_tapped"
        }
```

[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/mira-2111110003)

**Comment by lobontiumira** [comment]
Verified as fixed on the latest Focus Nightly 95.0 (build 353160913 GV 96.0a1-20211111045525) with Google Pixel (Android 10).
[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/mira-2111120001)

```
{
          "timestamp": 0,
          "category": "app_shortcuts",
          "name": "erase_open_button_tapped",
          "extra": {
            "opened_tabs": "1"
          }
        }
```


```
{
          "timestamp": 0,
          "category": "app_shortcuts",
          "name": "just_erase_button_tapped",
          "extra": {
            "opened_tabs": "1"
          }
        }
```

```
{
          "timestamp": 37,
          "category": "recent_apps",
          "name": "app_removed_from_list"
        }
```
